internal/server/http/imposter.go

```diff
@@ -9,7 +9,26 @@ import (
 	"time"
 )
 
-const imposterExtension = ".imp.json"
+// ImposterType allows to know the imposter type we're dealing with
+type ImposterType int
+
+const (
+	jsonImposterExtension = ".imp.json"
+	yamlImposterExtension = ".imp.yml"
```

Comment:
According to [YAML's official FAQs](https://yaml.org/faq.html), the official extension is `.yaml`.

However, I know `.yml` is also [widely used](https://stackoverflow.com/questions/21059124/is-it-yaml-or-yml).

So, I'd suggest you to add support for both file extensions.